modules/EnsEMBL/Web/ConfigPacker.pm

```diff
@@ -1934,5 +1937,10 @@ sub _munge_species_url_map {
   $multi_tree->{'ENSEMBL_SPECIES_URL_MAP'} = \%species_map;
 }
 
+sub is_collection {
+  my ($self, $db_name) = @_;
+  my $database_name = $self->tree->{'databases'}->{'DATABASE_CORE'}{'NAME'};
```

Comment:
use $db_name instead of hardcoding DATABASE_CORE

Reply:
Done - but will default $db_name to 'DATABASE_CORE' if undef, so that there's no chance of breaking existing code that might omit the variable.